src/login/Template.tsx

```diff
@@ -132,12 +77,24 @@ export default function Template(props: TemplateProps<KcContext, I18n>) {
                     {msg("requiredFields")}
                   </span>
                 </div>
-                <div className="col-md-10">
-                  <h1 id="kc-page-title">{headerNode}</h1>
+                <div className="headerLogo">
+                  <img
+                    src={`${url.resourcesPath}/img/logo_and_text_light_mode.svg`}
```

Comment:
all of these images are broken for me locally. how can I get them to render properly?

Reply:
not sure why they wouldn't load locally but just to be safe I updated the images to pull from S3